templates/logrotate.j2

```diff
@@ -0,0 +1,13 @@
+{{ prom2teams_logs_folder }}/uwsgi.log {
```

Comment:
I would use a wildcard, so any log in that folder would be rotated :D

Reply:
The issue is that there are others that don't need to be rotated by logrotate :)